### ISSUE TITLE
POP3 Retriever Should Not Force SSL Verify None

### ISSUE THREAD

**Issue by carsonreinke** [mikel/mail#577]
Currently, the POP3 retriever does this whenever SSL is enabled:

``` ruby
pop3.enable_ssl(OpenSSL::SSL::VERIFY_NONE) if settings[:enable_ssl]
```

(https://github.com/mikel/mail/blob/master/lib/mail/network/retriever_methods/pop3.rb#L129)

Should this not be like SMTP delivery method which allows for `:openssl_verify_mode` setting?

**Comment by jeremy** [comment]
Yes it should be! And VERIFY_NONE should certainly not be the default.
